pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -103,16 +103,16 @@
 pkgid_to_location_href = collections.defaultdict(dict)
 
 
-def store_metadata_for_mirroring(repo, dl_result, relative_path):
+def store_metadata_for_mirroring(repo, path, relative_path):
     """Used to store data about the downloaded metadata for mirror-publishing after the sync.
 
     Args:
         repo: Which repository the metadata is associated with
-        dl_result: The DownloadResult from downloading the metadata
+        path: The path to the metadata file
```

Comment:
Should we name it more specifically? metadata_file_path? not sure. Maybe just path is fine.

Reply:
I'll change this.